site/content/docs/howto/supervisor/configure-supervisor-with-github.md

```diff
@@ -77,39 +76,45 @@ metadata:
   name: github-enterprise
 spec:
   githubAPI:
+    # This field is only required when using GitHub Enterprise Server.
     # Only the hostname or IP address and optional port, without the protocol.
     # Pinniped will always use HTTPS.
     host: github.enterprise.tld
     tls:
+      # This field is only required when using GitHub Enterprise Server.
```

Comment:
I didn't specify that this was required because that depends on how the image containing Pinniped is set up. For GHES the image could be preloaded with the right CAs (which means this field is not required). For github.com the image might not have any CAs at all (which means this field is required).

Reply:
Good point. Updated.